config/default.json

```diff
@@ -0,0 +1,47 @@
+{
+  "openapiConfig": {
+    "filePath": "./openapi3.yaml",
+    "basePath": "/docs",
+    "rawPath": "/api",
+    "uiPath": "/api"
+  },
+  "telemetry": {
+    "logger": {
+      "level": "info",
+      "prettyPrint": false
+    }
+  },
+  "server": {
+    "port": "8001",
```

Comment:
True, the reason it was on a different port was because in order to check the connectivity of everything - 
Geocoding -> feedback-api -> geocoding-enrichment, I couldn't put everything on port 8080